src/components/App.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Header from './Header';
+import '../styles/App.scss';
+import AppWrapper from "./AppWrapper";
+
+const App = () => {
+    return (
+        <div className="app">
+            <Header/>
+            <AppWrapper/>
```

Comment:
zvazil bych lepsi naming, protoze z `AppWrapper` mam tendenci si myslet, ze to obaluje celou `App` komponentu, zatimco tady je to uvnitr.

Klidne muze byt obycejne `Content`, z hlediska SEO by to melo byt v pohode, pac mas `HEADER`, `sekce` a `footer` :)